packages/itwin/geo-tools/src/GeoToolsUiItemsProvider.tsx

```diff
@@ -2,33 +2,31 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import { StageUsage, ToolbarHelper, ToolbarOrientation, ToolbarUsage } from "@itwin/appui-react";
+import { ToolbarItemUtilities, ToolbarOrientation, ToolbarUsage } from "@itwin/appui-react";
 import type { ToolbarItem, UiItemsProvider } from "@itwin/appui-react";
 
 import { GeoToolsItemDef } from "./GeoToolsItemDef";
 
 export class GeoToolsAddressSearchProvider implements UiItemsProvider {
   public readonly id = "GeoToolsAddressSearchProvider";
 
-  public  provideToolbarItems(
-    _stageId: string,
-    stageUsage: string,
-    toolbarUsage: ToolbarUsage,
-    toolbarOrientation: ToolbarOrientation
-  ): ToolbarItem[] {
-    if (
-      stageUsage === StageUsage.General &&
-      toolbarUsage === ToolbarUsage.ViewNavigation &&
-      toolbarOrientation === ToolbarOrientation.Horizontal
-    ) {
-      return [
-        ToolbarHelper.createToolbarItemFromItemDef(
-          70,
-          GeoToolsItemDef.geoAddressSearchButtonItemDef,
-        ),
-      ];
-    }
-
-    return [];
+  public getToolbarItems(): ToolbarItem[] {
+    const horizontalLayoutOverride = {
+      standard: {
+        orientation: ToolbarOrientation.Horizontal,
+        usage: ToolbarUsage.ViewNavigation,
+      },
+    };
+    GeoToolsItemDef.geoAddressSearchButtonItemDef
```

Comment:
whats this line?

Reply:
Uggh - should have been removed.  Compiler didn't complain.